applications/rubintv/templates/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: {{ template "rubintv.fullname" . }}
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    all:
+      - "exec:notebook"
```

Comment:
This definitely shouldn't be `exec:notebook`. I'm not sure off-hand what it should be to read embargo data. We may need a separate scope for that. If everyone at the USDF can read embargo data, then maybe `read:image` is fine, but I'm not sure that's the case.

Reply:
I'll change to `read:image` for now and ask about embargo data on SLAC.  Merlin thought it was "anyone who can get an RSP notebook" hence this value.

Reply:
KT says `read:image` should be fine for the time being.